app.js

```diff
@@ -18,6 +18,8 @@ app.patch('/api/reviews/:review_id', patchVotes)
 
 app.get('/api/users', getUsers)
 
+app.delete('/api/comments/:comment_id', deleteComment)
+
 app.use((err,req,res,next)=>{
```

Comment:
This isn't directly related to this ticket but it may be worth moving the error handling functions out to a seperate file to keep your app.js file as minimal as possible.